src/content/app/tools/blast/types/parsedInputSequence.ts

```diff
@@ -18,3 +18,5 @@ export type ParsedInputSequence = {
   header?: string;
   value: string;
 };
+
+export type submittedSequence = ParsedInputSequence & { id: number };
```

Comment:
Please start type names with a capital letter.

Reply:
Also, I wonder if the declaration of this type more naturally belongs in the `blastResultsSlice.ts` file. Submitted sequences only exist around and after a submission. But I don't have a strong opinion about this.